main/createUiDefinition.json

```diff
@@ -65,7 +65,7 @@
         "toolTip": "Pass Jointoken.",
         "constraints": {
           "required": true,
-          "regex": "^[a-zA-Z0-9_-]{44}$",
+          "regex": "^[a-zA-Z0-9_-]{44,44}$",
           "validationMessage": "Insert validate Jointoken."
```

Comment:
should we say just `Invalid Jointoken` ?  or `Insert valid Jointoken`

Reply:
Good catch. Fixed typo.